src/main/java/com/epam/machine/repository/ClientRepositoryImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.machine.repository;
+
+import com.epam.machine.entity.Client;
+import lombok.Builder;
+import lombok.Data;
+
+import java.sql.*;
+
+@Data
+public class ClientRepositoryImpl implements ClientRepository{
+    final static String dataBaseUrl = "jdbc:postgresql://localhost:5432/Car";
+    final static String jdbcDriver = "org.postgresql.Driver";
+
+    final static String admin = "Admin";
+    final static String password = "qwerty";
+
+    @Builder
+    @Override
+    public Client get(int id) throws ClassNotFoundException,SQLException {
+        Class.forName(jdbcDriver);
+
+        String fullName = "";
+        String passport = "";
+        String driverCard = "";
+        String phoneNumber = "";
+        String eMail = "";
+        String login = "";
+        String pasword = "";
+
+        ResultSet resultSet;
+        try(Connection connection = DriverManager.getConnection(dataBaseUrl,admin,password);
+        Statement statement = connection.createStatement()) {
+            String sql = "SELECT * FROM driver WHERE driver.id = " + id + ";";
+            resultSet = statement.executeQuery(sql);
+            while (resultSet.next()) {
+                fullName = resultSet.getString(2);
+                passport = resultSet.getString(3);
+                driverCard = resultSet.getString(4);
+                phoneNumber = resultSet.getString(5);
+                eMail = resultSet.getString(1);
+                login = resultSet.getString(6);
+                pasword = resultSet.getString(7);
+            }
+        }
+
+        return Client.builder()
```

Comment:
I think you can build Client inside while block, so you will get rid of 22-28 rows and your return will be `return client;` which is more cleaner

Reply:
I meant smth like that
```
    @Override
    public Client get(int id) throws ClassNotFoundException, SQLException {
        Class.forName(JDBC_DRIVER);

        try (Connection connection = DriverManager.getConnection(DATA_BASE_URL, ADMIN, PASSWORD);
             PreparedStatement statement = connection.prepareStatement(
                     "select * from driver where driver.id = ?")) {

            statement.setInt(1, id);
            ResultSet resultSet = statement.executeQuery();
            resultSet.next();

            return Client.builder()
                    .id(id)
                    .fullName(resultSet.getString(1))
                    .passport(resultSet.getString(2))
                    .driverCard(resultSet.getString(3))
                    .phoneNumber(resultSet.getString(4))
                    .eMail(resultSet.getString(5))
                    .password(resultSet.getString(6))
                    .build();
        }
    }
```
there's no need to define Strings and then set them to Client instance, also `PreparedStatement` can help you to avoid long ugly sql queries, and for me it more readable, you can try it or if you feel that `Statement` is your choice, let it be